.gitignore

```diff
@@ -7,3 +7,6 @@
 /npm-debug.log
 /test.js
 /test/fixtures/espree-v8/node_modules
+.yarn
+.yarnrc.yml
+yarn.lock
```

Comment:
I use yarn locally.
To prevent accidental commit of 80+ files, I ignored these for now.
Can revert it when out of `draft`